herbert_core/utilities/classes/@object_lookup/private/parse_sz_repmat.m

```diff
@@ -0,0 +1,57 @@
+function sz_repmat_out = parse_sz_repmat (sz_repmat, n_objArr, ntot_objArr)
+% Check that the repmat option for implicitly expanding the stored arrays in an
+% object_lookup object is valid. Throw an error if not.
+%
+%   >> sz_repmat_out = parse_sz_repmat (sz_repmat, n_objArr, ntot_objArr)
+
+% Make sz_repeat a cell array for convenience
+if ~iscell(sz_repmat)
+    sz_repmat = {sz_repmat};    
+end
+
+% Check input is not empty
+nsz_repmat = numel(sz_repmat);
+if nsz_repmat==0
+    error('HERBERT:object_lookup:invalid_argument', ...
+        'If it has been given, the set of repeat sizes cannot be empty')
+end
+
+% Check each input is a valid single expansion argument to repmat
+is_size_vector = @(x)(isnumeric(x) && isrow(x) && numel(x)>=1 && ...
+    all(x>=0) && all(rem(x,1)==0));
+if ~all (cellfun(@(x)is_size_vector(x), sz_repmat), 'all')
+    error('HERBERT:object_lookup:invalid_argument', ...
+        'Repeat sizes must all be an integer>=0 or a valid Matlab array size')
+end
```

Comment:
See `isindex` function `herbert_core/utilities/general/isindex.m`

Reply:
Not quite the same as what I want. isindex will pass if the input is a column, or empty 1x0 or 0x1, or logical, whereas I want a numeric array length at least one of +ve integers